py/torch_tensorrt/dynamo/conversion/converter_utils.py

```diff
@@ -46,26 +47,53 @@ def get_node_name(node: torch.fx.Node) -> str:
 
 
 def dynamic_unsupported(node: torch.fx.Node) -> bool:
+    """Validates that a node has no dynamic args, kwargs, or outputs"""
+    return _dynamic_unsupported(node=node)
+
+
+def dynamic_unsupported_with_args(
+    arg_positions_to_check: Optional[List[int]] = None,
+) -> Callable[[torch.fx.Node], bool]:
```

Comment:
May need to add `from typing import Callable` at the top of the file

Reply:
Thanks for pointing it out. It was issue in rebasing. Addressed it,